projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -719,6 +727,13 @@ public static function escape_and_sanitize_field_value( $value ) {
 			return '';
 		}
 
+		// Check if this is a JSON-encoded file upload and extract the filename if it is.
+		$json_decoded = json_decode( $value, true );
+		if ( $json_decoded && isset( $json_decoded['name'] ) && isset( $json_decoded['file_id'] ) ) {
```

Comment:
we do this all the time, maybe we should add a helper, or add another field to easily identify it is a file

Reply:
there's another method that does this already?